src/components/Message/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import Button from '../Button'
+import ButtonGroup from '../ButtonGroup'
+import Input from '../Form/Input'
+import styles from './styles.less'
+
+export default class Message extends Component {
```

Comment:
@jaredreich @asabhaney what do we think about maybe calling this Dialog instead? (the common term used by a lot of UI frameworks)